### PR TITLE
add expo-font package, add Rubik Regular and Rubik-Medium fonts

### PR DESCRIPTION
**Pull request by bohdanprog**
- add expo-font package
- add Rubik Regular and Rubik Medium fonts.

### CODE REVIEW

**Review by achou11 — APPROVED** [review]
couple of suggestions but otherwise changes make sense to me!

**Comment by achou11** [comment]
Based on what's detailed in https://openfontlicense.org/how-to-use-ofl-fonts/, might also make sense to include the font's license file (if you downloaded from Google fonts, it's the file named `OFL.txt`). Can rename to something more obvious like `RUBIK_LICENSE.txt`

File: package.json [hunk 1]
```diff
@@ -51,9 +51,11 @@
     "cheap-ruler": "^3.0.2",
     "eventemitter3": "^5.0.1",
     "expo": "~50.0.8",
+    "expo-av": "^13.10.6",
```
[the remark this answers is not in the document]
Comment (bohdanprog) [reply package.json:54]: @achou11 
IDK why but you have a different opinion about it :D
[here](https://github.com/digidem/comapeo-mobile/pull/375#discussion_r1616269698) is why it looks on that way

[the remark this answers is not in the document]
Comment (achou11) [reply package.json:54]: ah funny 😄  - i would default to what `expo install` does, which is pin to patch versions.

[the remark this answers is not in the document]
Comment (bohdanprog) [reply package.json:54]: me too :D

[the remark this answers is not in the document]
Comment (bohdanprog) [reply package.json:54]: okay I'm gonna change that :)

**Comment by bohdanprog** [comment]
@achou11  I fixed everything, if it's okay we can merge that :)

**Pull request merged**